test/topology_experimental_raft/test_cdc_generation_data.py

```diff
@@ -0,0 +1,19 @@
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.topology.util import check_token_ring_and_group0_consistency
+
+import pytest
+
+@pytest.mark.asyncio
+async def test_send_data_in_parts(manager: ManagerClient):
+    config = {
+        'commitlog_segment_size_in_mb': 2
+    }
+
+    first_server = await manager.server_add(config=config)
+
+    async with inject_error(manager.api, first_server.ip_addr, 'cdc_generation_mutations_replication'):
+        for _ in range(3):
```

Comment:
Isn't it enough to boot a single node after doing the injection? If it is, then let's do only that so that the test doesn't run unnecessarily long.

Reply:
Good point, changed.